indico/modules/events/contributions/controllers.py

```diff
@@ -121,3 +123,12 @@ def _process(self):
                        "{count} contributions have been deleted.", deleted_count)
               .format(count=deleted_count), 'success')
         return jsonify_data(html=_render_contribution_list(self.event_new))
+
+
+class RHContributionPersonList(RHContributionPersonListMixin, RHManageContributionsActionsBase):
+    """List of persons in the contribution"""
+
+    @property
+    def _membership_filter(self):
+        contribution_ids = {contrib.id for contrib in self.contribs}
+        return Contribution.id.in_(contribution_ids)
```

Comment:
In case `in_()` accepts any iterable:

    return Contribution.id.in_(contrib.id for contrib in self.contribs)